core/src/main/scala/kafka/consumer/FetchRequestAndResponseStats.scala

```diff
@@ -34,6 +34,7 @@ class FetchRequestAndResponseMetrics(metricId: ClientIdBroker) extends KafkaMetr
 
   val requestTimer = new KafkaTimer(newTimer("FetchRequestRateAndTimeMs", TimeUnit.MILLISECONDS, TimeUnit.SECONDS, tags))
   val requestSizeHist = newHistogram("FetchResponseSize", biased = true, tags)
+  val throttleTimeHist = newHistogram("FetchRequestThrottleTimeMs", biased = true, tags)
```

Comment:
If you were to use a timer, the timer would give you both a rate as well as a histogram. Do you think that would be better? i.e., it will give you the request throttle rate (i.e., x throttled requests/sec) as well as the throttle time histogram (for those requests that were throttled)